x-pack/plugins/cloud_security_posture/public/common/hooks/use_navigate_findings.ts

```diff
@@ -46,7 +46,7 @@ export const useNavigateFindings = () => {
 export const useNavigateFindingsByResource = () => {
   const history = useHistory();
 
-  return (query?: Query['query']) => {
+  return (query: Query['query'] = {}) => {
```

Comment:
quick fix that navigates to findings with empty filters which override previous ones instead of persisting them. 

it's kinda hacky because it moves the responsibility of resetting the query/filters to the consumer which shouldn't really care about it. we will improve this in 8.7.